.github/scripts/bash/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+BASH_PATH=${BASH_PATH:-~/$TARGET/bash}
```

Comment:
Is this a new convention to apply to other packages?

Reply:
Yes, to separate builds for different hosts.